src/windowsservercore/2004/helix/webassembly/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-amd64
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-amd64-20200904200251-272704c
```

Comment:
It was right the way you had it before.  You just need to add a local tag like the following:
```json
"tags": {
  "windowsservercore-2004-helix-amd64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
  "windowsservercore-2004-helix-amd64": {
    "isLocal": true
  }
}
```

at this point: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/441/files#diff-ffa5b716b5a57837f7929dfcca4b4dfdeb97210a7fd5a12d2f1978846d6f1743R1086-R1088